expirationd.lua

```diff
@@ -502,10 +509,7 @@ local function expirationd_task_stats(name)
     return retval
 end
 
--- kill task
-local function expirationd_kill_task(name)
-    return get_task(name):kill()
-end
+
```

Comment:
Do not get this change

Reply:
I removed duplicate code from
https://github.com/tarantool/expirationd/blob/6c7ff791ce77a176540ca7a7233e889788bbbd72/expirationd.lua#L485-L487